install_config/persistent_storage/persistent_storage_azure_file.adoc

```diff
@@ -42,24 +42,73 @@ $ /usr/sbin/setsebool -P virt_use_samba on
 $ /usr/sbin/setsebool -P virt_sandbox_use_samba on
 ----
 
+. Type the `mount` command to check `dir_mode` and `file_mode` permissions, for example:
++
+----
+$ mount
+----
+
+If the `dir_mode` and `file_mode` permissions are set to `0755`,  change the
+default value `0755` to `0777` (or `0775`). This manual step is required because
+the default `dir_mode` and `file_mode` permissions changed from `0777` to `0755`
+in {product-title} 3.9. The following examples show configuration files with the 
+changed values.
+
+.Example persistent volume configuration file
+----
+# cat azf-pv.yml 
```

Comment:
I'd remove the cat commands and change the source type to yaml. (It doesn't look like you use these filenames in this topic.)